### ISSUE TITLE
Xcode 10.2 Beta - dyld: Symbol not found: _$SBOWV

### ISSUE THREAD

**Issue by chrisspankroy** [Estimote/iOS-Proximity-SDK#39]
I'm having an almost identical issue as #30 . After the app compiles and is installed on the device, it immediately crashes with the output:
```dyld: Symbol not found: _$SBOWV
  Referenced from: ~/Library/Developer/CoreSimulator/Devices/48B061D3-3B73-4447-BED3-0BBC586D2BF9/data/Containers/Bundle/Application/266D0BEB-7971-44E8-8669-13220DBD7995/MyApp.app/Frameworks/EstimoteProximitySDK.framework/EstimoteProximitySDK
  Expected in: ~/Library/Developer/CoreSimulator/Devices/48B061D3-3B73-4447-BED3-0BBC586D2BF9/data/Containers/Bundle/Application/266D0BEB-7971-44E8-8669-13220DBD7995/MyApp.app/Frameworks/libswiftCore.dylib
 in ~/Library/Developer/CoreSimulator/Devices/48B061D3-3B73-4447-BED3-0BBC586D2BF9/data/Containers/Bundle/Application/266D0BEB-7971-44E8-8669-13220DBD7995/MyApp.app/Frameworks/EstimoteProximitySDK.framework/EstimoteProximitySDK
```
This version of Xcode is the first one (I believe) that uses Swift 5. Looking through past issues in this repo, I think this issue has happened whenever there is a new Swift release? I installed the SDK via CocoaPods and I'm using version 1.2.1.

**Comment by speakerbug** [comment]
I'm also having this same issue.

**Comment by chrisspankroy** [comment]
This issue is fixed for me in the release build of Xcode 10.2
